### ISSUE TITLE
Help me add the Linux and MacOS implementations

### ISSUE THREAD

**Issue by udit-001** [udit-001/daily-wallpaper#1]
This project currently only works on Windows, but I want to add support for other systems too.
Kindly add a different file for other systems.

**Comment by davistardif** [comment]
Check out this package for Linux that already has most (all?) of the features you're looking for
[Variety](http://peterlevi.com/variety/)

**Comment by udit-001** [comment]
Does it have a python package so that I can use it in my own project?

**Comment by syre** [comment]
@davistardif i think he's looking for help to implement much of the same functionality variety has for his daily-wallpaper

**Comment by udit-001** [comment]
Yes @davistardif that's exactly what I need help for, since I do not have a Linux or MacOS system with me, so I can't implement and check them myself.
